addon/components/array-container.js

```diff
@@ -134,6 +206,55 @@ export default Component.extend(PropTypeMixin, {
       this.notifyParentOfNewItem(newItem, index)
     },
 
+    onChange (bunsenId, value) {
+      const autoAdd = this.get('cellConfig.item.autoAdd')
+      const clearingValue = [undefined, null, ''].indexOf(value) !== -1
```

Comment:
Seeing as it is only 3 characters shorter I prefer the native approach that doesn't depend on lodash and requires one less function call.